lua/nvim_comment.lua

```diff
@@ -13,10 +13,16 @@ M.config = {
   -- Normal mode mapping left hand side
   line_mapping = "gcc",
   -- Visual/Operator mapping left hand side
-  operator_mapping = "gc"
+  operator_mapping = "gc",
+  -- Hook function to call before commenting takes place
+  hook = nil
 }
 
 function M.get_comment_wrapper()
+  if type(M.config.hook) == 'function' then
```

Comment:
I'd actually suggest moving this to line 97 before get_comment_wrapper is set
whilst for this use case it's probably to be used to override commentstring value, it could be for somehitng else like changing the comment highlight value.

Reply:
Good point, did not consider other use cases. Have moved to as instructed.

Reply:
:heart: